### PR TITLE
pce 20160308 (devel)

### PR DESCRIPTION
**Pull request by tomyun**
### All Submissions:

- [X] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

### New Formulae Submissions:

- [X] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [X] Have you built your formula locally prior to submission with `brew install <formula>`?

### CODE REVIEW

File: Library/Formula/pce.rb [hunk 1]
```diff
@@ -7,8 +7,9 @@ class Pce < Formula
   head "git://git.hampa.ch/pce.git"
 
   devel do
-    url "http://www.hampa.ch/pub/pce/pre/pce-20140222-4b05f0c.tar.gz"
-    sha256 "44edaf071bb6840b6b3336174d528ff10c4dba8cb38194f0289fda81ac34f57f"
+    url "http://www.hampa.ch/pub/pce/pre/pce-20160308-72f1e10.tar.gz"
+    version "0.2.2+20160308"
```
Comment (dunn) [inline Library/Formula/pce.rb]: This is probably going to be computed as a lower version than the previous, so we should probably leave it scanned from the URL.

Comment (tomyun) [inline Library/Formula/pce.rb]: They seem to alternate version schemes between stable and unstable versions. For example, they had `20130218`  between `0.2.1` and `0.2.2`. I know prefixing version in this way may break some existing users for now, but the current devel version will prevent upgrading to stable versions later anyways. Another thing is that version scanning didn't work correctly with their filenames. The version was supposed to be `20140222`, but was set to `20140222-4` due to the following hash part.

Comment (dunn) [inline Library/Formula/pce.rb]: Even if upstream alternates their version scheme, we can always use the date.  Would that work?

Comment (tomyun) [inline Library/Formula/pce.rb]: Do you mean that we would use the date for even stable versions? I thought the other way around. We could always use numerics (0.x.x) and optionally add date for unstable release. The current formula already uses stable version `0.2.2`.

Comment (tomyun) [inline Library/Formula/pce.rb]: Or does brew distinguish versions between stable and devel? i.e. no upgrade from devel to stable even if higher stable versions available?

Comment (dunn) [inline Library/Formula/pce.rb]: > Or does brew distinguish versions between stable and devel? i.e. no upgrade from devel to stable even if higher stable versions available?

Yep!

Comment (tomyun) [inline Library/Formula/pce.rb]: A new thing that I just learned! Okay, the formula is updated to conform the current scheme.

**Comment by dunn** [comment]
Merged as Homebrew/homebrew-core@4c859fc0786376f66bb5e968fc2cb9e08ebfd2a0, thanks again!

**Pull request closed without merging**